composer/7/Dockerfile

```diff
@@ -5,19 +5,13 @@
 #
 
 # Pull base image
-FROM phalconphp/php:7
+FROM phalconphp/php:alpine-php7-min
 
 MAINTAINER Serghei Iakovlev <serghei@phalconphp.com>
+ENV COMPOSER_ALLOW_SUPERUSER 1
+ENV COMPOSER_HOME /composer
 
-# Set environment variables
-ENV COMPOSER_HOME=/root/composer \
-    PATH=$COMPOSER_HOME/vendor/bin:$PATH \
```

Comment:
do we really need that? I don't think we do, composer's executable is already available, and composer's global installations aren't meant to be executed inside a docker container. According to official composer, we should run like this:
```
docker run --rm --interactive --tty \
    --volume $PWD:/app \
    composer install
```